contrib/testsuites/run-kvm-unit-test.sh

```diff
@@ -21,9 +21,9 @@
 WILDCARD="*"
 while [ true ]; do
     case $1 in
-        "--endian")
+        --configure-args*)
```

Comment:
Do you have a special reason for allowing extra characters after the `--configure-args`? I don't think `--configure-args-foo` should be accepted (basically I don't think the tailing `*` should be there, but if you have a special reason for it, please let me know).